### PR TITLE
Add disability history to pensions form

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2224

This screen has been whittled down some and so there are some schema changes that I'm deferring until we get it approved by stakeholders.

screen shot 2017-05-31 at 12 35 36 pm
screen shot 2017-05-31 at 12 35 48 pm

### CODE REVIEW

File: src/js/pensions/config/form.js [hunk 1]
```diff
@@ -133,17 +138,62 @@ const formConfig = {
...
+                viewField: DisabilityField
+              },
+              items: {
+                name: {
+                  'ui:title': 'Disability'
+                },
+                disabilityStartDate: dateUI('Date disability began')
+              }
+            },
+            // TODO: update schema with this field if stakeholders approve
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Is this TODO done? Could this comment be removed? Or is this change in the process of being approved?

File: src/js/pensions/config/form.js [hunk 2]
```diff
@@ -133,17 +138,62 @@ const formConfig = {
...
+      }
+    },
     financialDisclosure: {
       title: 'Financial Disclosure',
       pages: {
         netWorth: {
           path: 'financial-disclosure/net-worth',
           title: item => `${item.veteranFullName.first} ${item.veteranFullName.last} net worth`,
           initialData: {
-            veteranFullName: {
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Why remove the veteranFullName initialData but not the spouseFullName initialData? Is this based on tests written so far?

**Review by cehsu — APPROVED** [review]
LGTM. Had some minor, non-essential questions.

File: src/js/pensions/config/form.js (hunk 2 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: It's based on the pages we've written so far. The applicant info page you added means we can add a veteran name, but we don't have a spouse page, so this is the only way to get the spouse financials page to show up.

File: src/js/pensions/config/form.js (hunk 1 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: It's in progress, should be resolved after our first stakeholder review (in the next couple weeks).

**Pull request merged**